activitysim/defaults/tables/skims.py

```diff
@@ -39,3 +39,17 @@ def sovmd_skim(nonmotskm_matrix):
 def sovpm_skim(nonmotskm_matrix):
     # FIXME use the right omx file
     return skim.Skim(nonmotskm_matrix, offset=-1)
+
+
+@sim.injectable(cache=True)
+def skims():
+    skims = skim.Skims()
+    # FIXME - this is reusing the same skim as all the different kinds of skims
+    for typ in ["SOV_TIME", "SOVTOLL_TIME", "HOV2_TIME",
```

Comment:
This is for later, but it would be good to define these at the top of a file or in a spec rather than inline in the code.  It would just be good to be super obvious about stuff like this that could change.